chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -53,7 +53,7 @@
   action :add_tenant
 end.run_action(:add_tenant)
 
-%x{keystone --os_username #{tempest_comp_user} --os_password #{tempest_comp_pass} --os_tenant_name #{tempest_comp_tenant} --os_auth_url #{keystone_settings["internal_auth_url"]} endpoint-get --service orchestration &> /dev/null}
+%x{keystone --insecure --os_username #{tempest_comp_user} --os_password #{tempest_comp_pass} --os_tenant_name #{tempest_comp_tenant} --os_auth_url #{keystone_settings["internal_auth_url"]} endpoint-get --service orchestration &> /dev/null}
```

Comment:
That is why I wrote into the commit message:
> since tempest is not used in production,
> and accessing services over the admin network,
> we do not need to verify certificates